internal/controller/controllers/agent_controller.go

```diff
@@ -703,6 +703,7 @@ func (r *AgentReconciler) updateInventory(log logrus.FieldLogger, host *models.H
 		log.WithError(err).Errorf("Failed to unmarshal host inventory")
 		return err
 	}
+	agent.Status.Hostname = inventory.Hostname
```

Comment:
What's the point of agent.Status.Hostname if it's the same as agent.Status.Inventory.Hostname?

Reply:
So what is https://issues.redhat.com/browse/MGMT-8337 about?

You want to remove the field?

Reply:
I remove the field from the CRD